data/ecosystems/s/starknet.toml

```diff
@@ -3,6 +3,7 @@ title = "Starknet"
 
 sub_ecosystems = [
   "10K Swap",
+  "Argent",
```

Comment:
Pull in the entire Argent sub_ecosystem rather then the specific repositories.

https://github.com/electric-capital/crypto-ecosystems/blob/047701e41c26b7e342dc4db8c17bd57e431a6255/data/ecosystems/a/argent.toml